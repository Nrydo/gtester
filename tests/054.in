8601794
772134901
459123534