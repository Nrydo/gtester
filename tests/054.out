9235042654408175